package/src/components/Message/MessageSimple/MessageStatus.tsx

```diff
@@ -120,9 +124,12 @@ const MemoizedMessageStatus = React.memo(
 export type MessageStatusProps = Partial<MessageStatusPropsWithContext>;
 
 export const MessageStatus = (props: MessageStatusProps) => {
-  const { message, readBy, threadList } = useMessageContext();
+  const { channel } = useChannelContext();
```

Comment:
Since we're only really using the channel to see if the `members` length is greater than 2, perhaps we could precalculate this elsewhere and just pass it to be used as an atomic value ? That way we won't have to worry about unstable references changing (nor probably need to run `useChannelContext()` like this downstream (which will trigger many changes). Having it here pre-memoization is definitely an improvement but we can still move it outside since we were anyway never passing it as a prop so it isn't breaking either.